less/textinput.less

```diff
@@ -36,6 +36,9 @@
     &.textinput-validation-error {
         border: 2px solid @validation-error-border-color;
     }
+    &::-ms-clear {
```

Comment:
I think it would be useful to include a brief comment here - perhaps with a link to the issue on github too